sync_boostr_data.py

```diff
@@ -0,0 +1,143 @@
+from dataclasses import dataclass
+import django
+import environ
+import logging
+import math
+import os
+import requests
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "consvc_shepherd.settings")
+django.setup()
+
+from consvc_shepherd.models import BoostrDeal, BoostrDealProduct, BoostrProduct
+
+@dataclass(frozen=True)
+class BoostrSyncConfig():
```

Comment:
Nit: empty parentheses are not necessary

```suggestion
class BoostrSyncConfig:
```